setup.py

```diff
@@ -598,6 +598,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'tzlocal>=1.4,<2.0.0',
     'unicodecsv>=0.14.1',
     'werkzeug<1.0.0',
+    'WTforms<2.3.0',
```

Comment:
```suggestion
    'WTforms<2.3.0',  # TODO: Remove after https://github.com/dpgaspar/Flask-AppBuilder/issues/1356 is fixed and released.
```
WDYT @potiuk ? I would like to keep track of those quick fixes / issues in other libraries we use. And remove those deps as soon as possible.